runtime/vm/resolvesupport.cpp

```diff
@@ -455,6 +455,26 @@ resolveStaticMethodRefInto(J9VMThread *vmStruct, J9ConstantPool *ramCP, UDATA cp
 	} else {
 		cpClass = J9_CLASS_FROM_CP(ramCP);
 		lookupOptions |= J9_LOOK_CLCONSTRAINTS;
+
+		if ( (cpClass != NULL) && (cpClass->romClass != NULL) ) {
+			if (isResolvedClassAnInterface) {
+				if ((J9CPTYPE_INTERFACE_STATIC_METHOD != J9_CP_TYPE(J9ROMCLASS_CPSHAPEDESCRIPTION(cpClass->romClass), cpIndex)) &&
```

Comment:
Please pull `J9_CP_TYPE(J9ROMCLASS_CPSHAPEDESCRIPTION(cpClass->romClass), cpIndex)` into a local so the code is easier to read.